### PR TITLE
Add Jupyter/Google Colab quickstart example for pypdf_table_extraction (Camelot)

### PR DESCRIPTION
**Pull request by alexfazio**
Hello,

I am following up regarding [#503](https://github.com/camelot-dev/camelot/pull/503).

Been working with Camelot in Google Colab and have created a quickstart-style example that I believe would be a valuable addition to the project's repository and documentation.

The example notebook demonstrates how to set up and use Camelot within the Google Colab environment, making it easier for users to get started with the library. It includes step-by-step instructions, code snippets, and explanations to guide users through the process.

I have attached the example notebook to this pull request. Please review it and consider including it in the project's documentation or examples section.

Thank you for your consideration. If you have any questions or suggestions for improvement, please let me know.

Best regards,

### CODE REVIEW

**Comment by bosd** [comment]
Thanks for opening this PR.
Can you please update it to point to this new repo / package name?

**Comment by stefan6419846** [comment]
To add on this: Could you please rename "PyPDF2" to "pypdf" as well? It seems like you are already pointing to the correct repository, but the "PyPDF2" package name is not maintained any more.

**Comment by alexfazio** [comment]
> To add on this: Could you please rename "PyPDF2" to "pypdf" as well? It seems like you are already pointing to the correct repository, but the "PyPDF2" package name is not maintained any more.

Hello,

I have updated the Colab notebook to reflect the requested changes, including renaming "PyPDF2" to "pypdf" and updating the dependencies accordingly. I have also tested everything to ensure it is functioning correctly.

You can review the latest commit here: [Latest Commit](https://github.com/alexfazio/pypdf_table_extraction/tree/238b18b4bd0834e6f8744845100f835732b5ef4c).

Please let me know if there's anything else needed before merging.

File: examples/camelot_quick_start_notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "bIKqmN_5jpJB"
+      },
+      "source": [
+        "# Dependencies Overview\n",
+        "\n",
+        "- [Camelot](https://github.com/camelot-dev/camelot)\n",
+        "- [pypdf](https://github.com/py-pdf/pypdf)\n",
+        "- [GhostScript](https://www.ghostscript.com/index.html)\n",
```
Comment (bosd) [inline examples/camelot_quick_start_notebook.ipynb:23]:
```suggestion
        "- [GhostScript](https://www.ghostscript.com)\n",
```

File: examples/camelot_quick_start_notebook.ipynb [hunk 2]
```diff
@@ -0,0 +1,413 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/alexfazio/pypdf_table_extraction/blob/main/camelot_quick_start_notebook.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```
Comment (bosd) [inline examples/camelot_quick_start_notebook.ipynb:10]: Is it possbile to have this link to the main repo?

**Review by bosd — CHANGES_REQUESTED** [review]
Thanks for all your work!! Much appreciated.

Code review & quick functional test.
Looks good to me.
Some minor suggestions made about the package name and references.

Please rename the file to `pypdf_table_extraction_quick_start_notebook.ipynb`

**Comment by bosd** [comment]
One addition.. small suggestion, non blocking.
Maybe it is possible to output to a df as in this example.
https://gist.github.com/vinayak-mehta/e5949f7c2410a0e12f25d3682dc9e873
As it provides a direct feedback. It is a but more user friendly, then having to browse to the outputs folder to see the result.

File: examples/camelot_quick_start_notebook.ipynb (hunk 2 above)
Comment (alexfazio) [reply examples/camelot_quick_start_notebook.ipynb:10]: I think this might only be possible once it is merged?

Not entirely sure

**Comment by alexfazio** [comment]
> Thanks for all your work!! Much appreciated.
> 
> Code review & quick functional test. Looks good to me. Some minor suggestions made about the package name and references.
> 
> Please rename the file to `pypdf_table_extraction_quick_start_notebook.ipynb`

I have implemented all your feedback except for the link to the main repository for the Colab button. 

Please double-check and let me know if you have any further comments.

**Comment by bosd** [comment]
@alexfazio Thanks for your prompt follow up.
Please check your commits. As the latest one seems to introduce some noise, as it also includes the output from the terminal.
Some of them are undo-ing the changes like the rename.

**Comment by bosd** [comment]
Good catch, let's add `.DS_Store` as well 😉

**Comment by bosd** [comment]
Closed by mistake. New pr in #231

**Comment by TaciteOFF** [comment]
Current Colab version provided in the example doesn't include Ghostscript.

**Comment by bosd** [comment]
Corrrect, Ghostscript is no longer a hard requirement :)

**Pull request closed without merging**